src/topology/metric_space/hausdorff_distance.lean

```diff
@@ -492,6 +492,25 @@ lemma inf_dist_image (hΦ : isometry Φ) :
   inf_dist (Φ x) (Φ '' t) = inf_dist x t :=
 by simp [inf_dist, inf_edist_image hΦ]
 
+/-! `inf_dist` as a function into `ℝ≥0`. -/
```

Comment:
Should this become a markdown heading?
```suggestion
/-! `### inf_dist` as a function into `ℝ≥0`. -/
```

Reply:
Ok. I also added headers elsewhere in the file.